<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QMainWindow" name="RegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registration</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LU ID#:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailTextField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter Email...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameTextField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter Name...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="luIdTextField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter LU ID#...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phoneTextField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter Phone#...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameTextField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter Username...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordTextField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Enter Password...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirmPasswordTextField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Confirm Password...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="idPicture">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="passwordErrorLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password Error</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
